collect_app/src/main/java/org/odk/collect/android/preferences/GeneralPreferencesFragment.java

```diff
@@ -82,22 +82,21 @@ public void onCreate(final Bundle savedInstanceState) {
     public boolean onPreferenceClick(Preference preference) {
         if (MultiClickGuard.allowClick(getClass().getName())) {
             PreferenceFragmentCompat basePreferenceFragment = null;
-            boolean adminMode = getArguments().getBoolean(INTENT_KEY_ADMIN_MODE, false);
             switch (preference.getKey()) {
                 case "protocol":
-                    basePreferenceFragment = ServerPreferencesFragment.newInstance(adminMode);
+                    basePreferenceFragment = new ServerPreferencesFragment();
```

Comment:
Are we not losing the `adminMode` argument here?

Reply:
No I just pass arguments from activity to fragments https://github.com/getodk/collect/pull/4014/files#diff-fc03fbca20bb6f86edfdee031f3509a4R74